.gatling/example.package.conf

```diff
@@ -0,0 +1,54 @@
+# Learn more about Configuration as Code: https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
+# This file is full deployment configuration file example.
+# Please note all these configurations, or even this file, are optional.
+
+gatling.enterprise.package {
+  # Consistent deployment using an existing package ID
+  # https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code/#consistent-deployment-with-id
+  # id = "00000000-0000-0000-0000-000000000000"
+  name = "My package name"
+  team = "My team name" # or ID with team = "00000000-0000-0000-0000-000000000000"
+  default {
+    simulation {
+      locations = [
+        {
+          name: "Europe - Paris",
+          size: 2,
+          weight: 30
+        },
+        {
+          name: "AP Pacific - Mumbai",
+          size: 2,
+          weight: 70
+        }
+      ]
+      parameters {
+        ignoreDefaults = false
+        systemProperties {
+          # System properties names should be surroinded by quotes
```

Comment:
```suggestion
          # System properties names should be surrounded by quotes
```

Reply:
I'm playing with the new PR comments integration in IDEA, it seems I failed :smile: